fundingmanager.go

```diff
@@ -487,6 +493,17 @@ func (f *fundingManager) Start() error {
 func (f *fundingManager) start() error {
 	fndgLog.Tracef("Funding manager running")
 
+	// Locate the RPCAcceptor within the ChainedAcceptor and start it if it exists.
+	openChanPredicate := f.cfg.OpenChannelPredicate
+	chainedAcceptor, _ := openChanPredicate.(*chanacceptor.ChainedAcceptor)
+	for _, acceptor := range chainedAcceptor.Acceptors {
+		rpcAcceptor, ok := acceptor.(*chanacceptor.RPCAcceptor)
+		if ok {
+			rpcAcceptor.Start()
```

Comment:
Shouldn't the RPC acceptor _already_ be started if it has been passed in at this point?